pyproject.toml

```diff
@@ -94,6 +94,8 @@ develop = [
 ]
 examples = [
   "salesforce-merlion<2.1",
+  "pycaret[analysis,models,tuner,parallel,test]==3.1.0"
+  "werkzeug==2.2.3"
```

Comment:
May I ask where exactly Werkzeug is needed? Is there a bit of Flask involved anywhere?

Reply:
Yep "unfortunately" it is in the pycaret analyses extension.
In general the analyses extension is not needed for this minimal example, however as soon as you want to have a plot or model stats. I thought maybe it's good to include it in the example therefore, as people are certainly needing this package - so they can simply take this example and build upon it.

Would you suggest to remove the analyses extension (and therefore flask)?

Reply:
I just don't like Werkzeug/Flask too much. It is nice in many details, but Werkzeug's router is just a tick too less capable. At least it is what I know from ~2 years ago.

However, I don't suggest to remove anything on purpose, if something needs it to run a web API. It's just c'est la vie ;].